### PR TITLE
bug-fix:fixing the bug of a new confirmation pop up when cancelling t…

### PR DESCRIPTION
**Pull request by victor0villca**
**Author**
 Victor Villca @victor0villca 
**Bug Fix Description**
This is the bug fix of the following bug: https://github.com/CampusDelSaber/hermes/issues/207#issue-1781628614
**Taiga**
https://tree.taiga.io/project/marcel-morales-hermes/issue/511

### CODE REVIEW

**Review by fabianromeroclaros — CHANGES_REQUESTED** [review]
My mistake.*

**Review by fabianromeroclaros — APPROVED** [review]
Issue solved.

**Review by Misax148 — APPROVED** [review]
Solved

File: app/src/main/java/com/isc/hermes/controller/PopUp/PopUpConfirmIncidentCanceling.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package com.isc.hermes.controller.PopUp;
+
+import android.annotation.SuppressLint;
+import android.app.Activity;
+import android.view.View;
+import com.isc.hermes.R;
+import com.isc.hermes.controller.IncidentFormController;
+
+public class PopUpConfirmIncidentCanceling extends PopUp{
```
Comment (VillalobosGarciaGaby) [inline app/src/main/java/com/isc/hermes/controller/PopUp/PopUpConfirmIncidentCanceling.java:14]: this class needs java docs

**Review by VillalobosGarciaGaby — APPROVED** [review]

File: app/src/main/java/com/isc/hermes/controller/PopUp/PopUpConfirmIncidentCanceling.java (hunk 1 above)
Comment (victor0villca) [reply app/src/main/java/com/isc/hermes/controller/PopUp/PopUpConfirmIncidentCanceling.java:14]: This observation has been solved

**Pull request merged**